core/src/com/mygdx/battleship/OrientationType.java

```diff
@@ -0,0 +1,6 @@
+package com.mygdx.battleship;
+
+
+public enum OrientationType{
+    NORTH, EAST, SOUTH, WEST
```

Comment:
Why not add an opposite() method that returns the opposite direction so that you don't need to calculate it outside of the class in other places?